calc-front/components/Header.tsx

```diff
@@ -1,37 +1,44 @@
 import Image from "next/image";
 import Link from "next/link";
-import Logo from "../public/logo.png";
+
 import CityAutoDetect from "./CityAutoDetect";
 import { menuItems } from "@/lib/menuItems";
 
 export const Header = () => {
   return (
-    <header className="flex justify-between bg-background_header/50 absolute top-0 left-0 right-0 py-4">
-      <div className="container">
-        <div className="flex items-center justify-between">
-          <Link href="/" className="flex items-center" prefetch={false}>
-            <Image
-              src={Logo}
-              alt="Logo"
-              width={235}
-              height={80}
-              className="w-[235px] h-20 mr-2 object-contain"
-            />
-          </Link>
-          <nav className="flex gap-4">
+    <header className="hidden lg:inline-block bg-background_header/50 absolute top-6 left-0 right-0 py-6 px-24 rounded-xlg">
+      <div className="flex items-center justify-between">
+        <Link
+          href="/"
+          className="flex items-center leading-none"
+          prefetch={false}
+        >
+          <Image
```

Comment:
logo should be image, not svg
svg is not visible for seo

Reply:
Change svg to png